node/src/command.rs

```diff
@@ -225,6 +225,17 @@ pub fn run() -> Result<()> {
             let collator_options = cli.run.collator_options();
 
             runner.run_node_until_exit(|config| async move {
+                let bootstrap_config = if config.role.is_authority() {
+                    let p2p_key = cli.run.p2p_key.ok_or("Could not find p2p key")?;
```

Comment:
```suggestion
                    let p2p_key = cli.run.p2p_key.ok_or("This node is configured as authority, so it will be used as Bootstrap node for Storage Provider & Collator network, but the key is missing. Set --p2p--key argument of the node.")?;
```

Let's try to make error messages which are shown to the user as concise and sufficient enough so they don't need to go to the source code to figure this one out.